model/stats/db.go

```diff
@@ -1114,18 +1266,49 @@ func makeSum(condition bson.M) bson.M {
 ///////////////////////////////////////////////////////////////////
 // Functions to access pre-computed stats documents for testing. //
 ///////////////////////////////////////////////////////////////////
+// Get the last updated hourly test stats document for a given project.
+func GetLatestHourlyTestDoc(projectID string, requester string, tasks []string) (*DbTestStats, error) {
+        return getLatestTestDoc(hourlyTestStatsCollection, projectID, requester, tasks)
+}
+
+// GetLatestDailyTestDoc find the last updated daily test stats document for a given project.
+func GetLatestDailyTestDoc(projectID string, requester string, tasks []string) (*DbTestStats, error) {
+        return getLatestTestDoc(dailyTestStatsCollection, projectID, requester, tasks)
+}
+
+func getLatestTestDoc(collection string, projectID string, requester string, tasks []string) (*DbTestStats, error) {
```

Comment:
This needs to be adjusted. it's not quite right (it may be looking up via display_name but the needs to take the execution_display_name into account too).